packages/gatsby-plugin-nginx/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
-  "include": ["src", "types"],
+  "include": ["src", "types", "test"],
```

Comment:
Can you just remove this bit. tsdx doesn't include this by default

Reply:
Without that, the types don't resolve in the test case

![image](https://user-images.githubusercontent.com/672405/119742796-e1bf9600-becb-11eb-9afe-9fe171ba4e36.png)

Reply:
This happens when I open VSCode from packages/gatsby-plugin-nginx. Openning from the root folder makes it work. 

Since I hope to move this plugin to the official gatsby-plugin-nginx in the future and have only one plugin for nginx, I'll leave this then. Thanks!